version/version.go

```diff
@@ -0,0 +1,44 @@
+package version
+
+import (
+	"fmt"
+
+	"github.com/taskcluster/taskcluster-cli/extpoints"
+)
+
+var VersionNumber = fmt.Sprintf("taskcluster (TaskCluster CLI) version %d.%d.%d", 1, 0, 0)
+
+func init() {
+	extpoints.Register("version", version{})
+}
+
+type version struct{}
+
+func (version) ConfigOptions() map[string]extpoints.ConfigOption {
+	return nil
+}
+
+func (version) Summary() string {
+	return "Prints the TaskCluster version."
+}
+
+func usage() string {
+	return `Usage:
+  taskcluster VERSION
```

Comment:
According to the [docopt documentation](https://github.com/docopt/docopt#usage-pattern-format), the arguments are specified as upper case words. This forced me to make the usage as `taskcluster VERSION` instead of `taskcluster version`. 

Is that all right or am I missing something here?